lib/providers/providers.dart

```diff
@@ -94,59 +90,57 @@ final providers = [
     create: (context) => TelosBalanceNotifier(),
     update: (context, http, balance) => balance!..update(http: http),
   ),
```

Comment:
`TransactionsNotifier` and `TelosBalanceNotifier `

I do not see we are using them in V2 nor do I know if they are really necessary.